foreman/data_refinery_foreman/surveyor/external_source.py

```diff
@@ -51,7 +63,8 @@ def handle_batch(self, batch: Batch, key_values: BatchKeyValue = None):
         batch.survey_job = self.survey_job
         batch.source_type = self.source_type()
         batch.status = BatchStatuses.NEW.value
-        batch.internal_location = None
+        batch.internal_location = (self.source_type() + "/"
+                                   + batch.accession_code + "/")
```

Comment:
Good question. If we're running a specific flavor of Linux within Docker, I am a bit less concerned but we should still be writing solid Pythonic code wherever possible. (What if we wanted to switch to a different OS in Docker, for instance? Loose coupling is good.) I guess the example of the root path is something I would suggest is a setting that should ideally come in via a settings file or, as you say, an environment variable.